packages/sdk/app-framework/src/plugins/common/metadata.ts

```diff
@@ -2,7 +2,7 @@
 // Copyright 2023 DXOS.org
 //
 
-import type { Plugin } from '../PluginHost';
+import { type Plugin } from '../PluginHost';
```

Comment:
interesting, you prefer the `type` keyword inside the braces?

Reply:
i just find it more consistent over time.